install.zasilkovna.php

```diff
@@ -598,5 +610,31 @@ private function createCronToken() {
 
         $model->updateConfig($config);
     }
+
+    /**
+     * @return void
+     */
+    private function removeBackendJoomlaMenuEntry()
+    {
+        if (version_compare(JVERSION, '4.0.0', '<')) {
+            return;
+        }
+
+        $db = JFactory::getDbo();
+        $alias = self::PLUGIN_ALIAS;
+        $clientId = 1;
+        $query = $db->getQuery(true)
+            ->delete($db->quoteName('#__menu'))
+            ->where([
+                $db->quoteName('alias') . ' = :alias',
+                $db->quoteName('client_id') . ' = :client_id',
```

Comment:
Proč je client_id vyčleněno do proměnné a published ne ? Není to vyčlenění do proměnné clientId zbytečné ?

Reply:
samostatné proměnné jsou potřeba pro $query->bind.
zjednodušil jsem to a na hodnotu sloupce published nebudeme brát ohled. a client_id musí být 1 (1=backend) vždy